ee/desktop/menu/menu_template.go

```diff
@@ -32,7 +38,37 @@ func (tp *templateParser) Parse(text string) (string, error) {
 		return "", fmt.Errorf("templateData is nil")
 	}
 
-	t, err := template.New("menu_template").Parse(text)
+	t, err := template.New("menu_template").Funcs(template.FuncMap{
```

Comment:
template thing is so cool

Reply:
I know! I underestimated their power and originally thought this would be a much more complex implementation.